components/dashboard/roommaps/RoomRen.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Seat from './Seat';
+import styled from 'styled-components';
+import Link from 'next/dist/client/link';
+
+const RoomRen = ({ selected }) => {
+  return (
+    <Wrapper>
+      {selected.map((selections) => {
+        return (
+          <>
+            <Link href={'/booking/' + selections.desk}>
+              <a>
+                <Seat name={selections.desk} key={selections.desk} />
+              </a>
+            </Link>
+          </>
+        );
+      })}
```

Comment:
You could use object destructuring on the selections parameter to you advantage here:
 ```suggestion
      {selected.map(({desk}) => {
        return (
          <Link href={'/booking/' + desk}>
            <a>
              <Seat name={desk} key={desk} />
            </a>
          </Link>
        );
      })}
```

Also so the fragment is superfluous :)